src/main/java/org/opentripplanner/middleware/auth/Auth0Connection.java

```diff
@@ -142,24 +142,17 @@ public static void checkUserIsAdmin(Request req, Response res) {
         }
     }
 
-    /**
-     * Check if the incoming user is an admin user
-     */
-    public static boolean isUserAdmin(Auth0UserProfile user) {
-        return user != null && user.adminUser != null;
-    }
```

Comment:
This check appears to have been replaced with `RequestingUser#isAdmin`. Are we certain that the `user` is never null (i.e., are we avoiding NPEs when `user.isAdmin()` is invoked)?

Reply:
I think this is fine. All API calls (except to /auth) call `Auth0Connection.checkUser` which in-turn calls `Auth0Connection.isValidUser`. This does a null pointer check on the requesting user object.